### DISCUSSION TITLE
Compiling data to specific memory location

### DISCUSSION THREAD

**Discussion by IanWitham** [jkotlinski/durexforth#493] [Q&A] [answered]
Hi, noob question here... Is it possible to compile data (custom characters, for example) to a specific memory location? I'm looking for an equivalent to `*=$2000` in assembler.

The only way I can think of currently is to use something like:
`header my-data $1E c, $1E c,  ... (many more bytes)`
and then copy that data to the correct location. Now I have "my-data" taking up memory in my program when it's no longer needed.

Or is this what the "marker" word is for?  (i.e. to delete the data from "my-word" onward once it's not needed).

**Comment by jkotlinski** [comment, marked as answer]
Hallo!
HERE is the same as * in assembly.
You can do e.g. ”$c000 to here” to target $c000.
Cheers

**Comment by IanWitham** [comment]
Thank-you!
